### DISCUSSION TITLE
Patch problem

### DISCUSSION THREAD

**Discussion by FitzroyWang** [Mr-MIBonk/M.I.B._More-Incredible-Bash#130] [General]
**I have a problem with the patch.
I have the software MHI2Q_CN_AUG22_P0915 with MU-0915.**
![IMG_8128](https://user-images.githubusercontent.com/126141239/220836081-661e6457-e650-456c-aa39-35656e88ef37.JPG)
**I followed the instructions but when patching, I got the following error at the end:**
![IMG_8096](https://user-images.githubusercontent.com/126141239/220836121-8759423c-8bcd-4207-87c0-c083b7b61ef2.jpg)
**NO Flashfiles there! I have checked some times, and I find the FW of patches is right, but the patch file name is a little different，
Search File is MU0915......-30DF7F200.ifs, but patch file name is MU0915......-24DF7F200.ifs**
![IMG_8127](https://user-images.githubusercontent.com/126141239/220836156-8e36a0af-8edd-4b26-96c7-b12d7e389d66.JPG)
**Can I rename this file name? or other solutions？
Sorry for my English, I'm translating from Chinese to English.
Many Thanks for answers.**

**Comment by harman-f** [comment]
Looks like your unit is alsread running some kind of patch.
